File: backend/spring-boot/src/main/java/com/runhwani/runmate/dto/response/marathon/MarathonResponse.java
package com.runhwani.runmate.dto.response.marathon;

import lombok.*;

import java.time.LocalDate;
import java.util.List;
import java.util.UUID;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class MarathonResponse {
    private UUID marathonId;
    private String name;
    private LocalDate date;
    private String location;
    private List<String> distance;
}

File: RunMate/data/src/main/java/com/D107/runmate/data/remote/response/manager/MarathonResponse.kt
package com.D107.runmate.data.remote.response.manager

import com.D107.runmate.data.remote.common.BaseResponse

data class MarathonResponse(
    val marathonId: String,
    val name: String,
    val date: String,
    val location: String,
    val distance: List<String>
) : BaseResponse